Src/Plugins/AzureDevOpsConfiguration/AzureDevOpsConfiguration.csproj

```diff
@@ -11,6 +11,12 @@
     <PackageId>Sarif.PatternMatcher.AzureDevOpsConfiguration</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MSBuildWarningsAsMessages>
+	    $(MSBuildWarningsAsMessages);MSB3277;MSB3030
```

Comment:
>	     [](http://example.com/codeflow?start=0&length=5)

mixing of tab and space characters

Reply:
same to all other csproj files

Reply:
Nice catch. I didn't have display of spaces/tabs turned on and didn't see that what I pasted from an external source had the tabs.
My setup doesn't add tabs just the paste from outside.

Tabs removed now.